hyper-boring/src/lib.rs

```diff
@@ -326,3 +326,12 @@ where
         }
     }
 }
+
+impl<T: fmt::Debug> fmt::Debug for MaybeHttpsStream<T> {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        match *self {
+            MaybeHttpsStream::Http(..) => f.pad("Http(..)"),
+            MaybeHttpsStream::Https(..) => f.pad("Https(..)"),
```

Comment:
Why not show the debug impl for the underlying T as well?

Anyway, this seems good overall - if you either add the underlying impl or remove the `T: Debug` bound I'm happy to merge this.

Reply:
Ah that slipped by. I only really needed `MaybeHttpsStream` to implement it.